<ui version="4.0" >
 <class>SettingsSSLWidget</class>
 <widget class="QWidget" name="SettingsSSLWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>SSL Certficate</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>If you are using SSL to connect to the server (this is typical), then you must put a valid PEM certificate in this box.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QTextEdit" name="certTextEdit" >
        <property name="font" >
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections/>
</ui>
